src/java.base/share/native/libjava/VM.c

```diff
@@ -42,6 +55,11 @@ Java_jdk_internal_misc_VM_latestUserDefinedLoader0(JNIEnv *env, jclass cls) {
 
 JNIEXPORT void JNICALL
 Java_jdk_internal_misc_VM_initialize(JNIEnv *env, jclass cls) {
+#if defined(WIN32)
+    /* Other platforms do this in check_version.c JNI_OnLoad. */
+    UT_JCL_JAVA_MODULE_LOADED(J9_UTINTERFACE_FROM_VM(((J9VMThread *) env)->javaVM));
+#endif /* defined(WIN32) */
```

Comment:
Is AIX not a special case as it is in jdk17?

If this works for Windows, it should work for all platforms. Why do we want to have a solution that varies by platform?

Reply:
It works for AIX jdk11 without the special case. JNI_OnLoad() should be called, and seems the best place to initialize the tracepoints. Doing it in a native is not ideal, as the tracepoints won't be enabled until after the native is called. This is why I limited it to Windows, which except for AIX jdk17 is the only platform that isn't calling JNI_OnLoad().